### ISSUE TITLE
Starting coco --worker: ValueError: r_home is None.

### ISSUE THREAD

**Issue by pavlovdog** [plan-net/core4#152]
I've completed all installation steps on my Ubuntu 18.04. After I'm trying to run the `coco --worker`, I'm getting the following error:

```
[ core4 ] pavlovdog@pavlovdog:~/Work/core4$ coco --worker
start worker [worker@pavlovdog]
2019-12-22 13:16:38,620 - INFO     [core4.queue.worker.CoreWorker/worker@pavlovdog] registered daemon
2019-12-22 13:16:38,622 - INFO     [core4.queue.worker.CoreWorker/worker@pavlovdog] enter phase [startup]
2019-12-22 13:16:38,674 - INFO     [core4.queue.worker.CoreWorker/worker@pavlovdog] cleanup removed [0] sys.lock records
2019-12-22 13:16:38,699 - INFO     [core4.service.introspect.main.CoreIntrospector/worker@pavlovdog] start registration
2019-12-22 13:16:38,830 - ERROR    [core4.service.introspect.main.CoreProject/worker@pavlovdog] encountered error with module [core4.queue.helper.job.r]
Traceback (most recent call last):
  File "/home/pavlovdog/Work/core4/.venv/lib/python3.6/site-packages/core4/service/introspect/main.py", line 191, in _import_module
    mod = importlib.import_module(name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/pavlovdog/Work/core4/.venv/lib/python3.6/site-packages/core4/queue/helper/job/r.py", line 9, in <module>
    import rpy2.robjects.packages as rpackages
  File "/home/pavlovdog/Work/core4/.venv/lib/python3.6/site-packages/rpy2/robjects/__init__.py", line 14, in <module>
    import rpy2.rinterface as rinterface
  File "/home/pavlovdog/Work/core4/.venv/lib/python3.6/site-packages/rpy2/rinterface.py", line 4, in <module>
    from rpy2.rinterface_lib import openrlib
  File "/home/pavlovdog/Work/core4/.venv/lib/python3.6/site-packages/rpy2/rinterface_lib/openrlib.py", line 21, in <module>
    rlib = _dlopen_rlib(R_HOME)
  File "/home/pavlovdog/Work/core4/.venv/lib/python3.6/site-packages/rpy2/rinterface_lib/openrlib.py", line 14, in _dlopen_rlib
    raise ValueError('r_home is None. '
ValueError: r_home is None. Try python -m rpy2.situation
2019-12-22 13:16:38,997 - INFO     [core4.service.introspect.main.CoreProject/worker@pavlovdog] loaded [core4] with [4] jobs, [2] api containers
2019-12-22 13:16:39,222 - INFO     [core4.service.introspect.main.CoreIntrospector/worker@pavlovdog] registered [0] jobs to schedule
2019-12-22 13:16:39,474 - INFO     [core4.queue.worker.CoreWorker/worker@pavlovdog] enter phase [loop]
```

Looks like something wrong with the R? How can I fix it?

**Comment by pavlovdog** [comment]
Well, looks like the issue was fixed after the R was installed (I've used [this guide](https://www.digitalocean.com/community/tutorials/how-to-install-r-on-ubuntu-18-04)). Could you clarify this point in the documentation, please?

**Comment by m-rau** [comment]
resolved in branch develop with commit cb48565ee98ad3998e78a25c52b0ea8756885fda
